master/getting-started/kubernetes/upgrade/delete.md

```diff
@@ -14,6 +14,9 @@ following conditions.
   
 - [A data migration attempt failed partway through, leaving the etcdv3 datastore
   with some, but not all of your etcvd2 data](#deleting-calico-data-from-etcdv3-after-a-partial-migration).
+
+- [You upgraded to Calico v3.0 but then downgraded to your previous version
```

Comment:
Since they upgraded from v2.6.4, then we know their previous version and can change "your previous version" to "v2.6.4"

Reply:
I think you should specify in this bullet text that this is for the "Kubernetes API datastore" (versus etcd)